src/App.tsx

```diff
@@ -9,39 +9,44 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
+
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
Verify that `useDispatch` and `useSelector` are correctly imported from 'react-redux'.